ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt

```diff
@@ -65,74 +65,74 @@ hideall "--sync--"
 200.0 "--sync--" MapEffect { flags: "00020001", location: "17" } window 200.0,0
 204.1 "--targetable--"
 210.3 "--sync--" StartsUsing { id: "9CFF", source: "Usurper of Frost" } window 210.3,0
-215.3 "Quadruple Slap" Ability { id: "9CFF", source: "Usurper of Frost" }
-219.4 "Quadruple Slap" Ability { id: "9D00", source: "Usurper of Frost" }
-224.5 "--sync--" Ability { id: "9CEF", source: "Usurper of Frost" }
+215.3 "Quadruple Slap 1" Ability { id: "9CFF", source: "Usurper of Frost" }
+219.4 "Quadruple Slap 2" Ability { id: "9D00", source: "Usurper of Frost" }
+224.5 "--jump south--" Ability { id: "9CEF", source: "Usurper of Frost" }
 228.8 "Mirror Image" Ability { id: "9CF4", source: "Usurper of Frost" }
 235.9 "Diamond Dust" Ability { id: "9D05", source: "Usurper of Frost" }
 239.0 "--untargetable--"
-244.9 "Axe Kick/Scythe Kick" Ability { id: ["9D0A", "9D0B"], source: "Oracle's Reflection" }
-245.8 "The House of Light" Ability { id: "9D0E", source: "Fatebreaker's Image" }
+244.6 "Axe Kick/Scythe Kick" Ability { id: ["9D0A", "9D0B"], source: "Oracle's Reflection" }
```

Comment:
Would it be worth shortening this to `Axe/Scythe Kick` in `replaceText`?

Reply:
Can't hurt.